spec/javascript/components/organisms/StatusModalWrapper.test.js

```diff
@@ -37,6 +37,26 @@ describe('StatusModalWrapper', () => {
     expect(wrapper.find(FormModal).prop('isOpen')).toBeFalsy()
   })
 
+  test('should display a subtitle if numApplicationsToUpdate > 1', () => {
+    wrapper = getWrapper({ numApplicationsToUpdate: 2 })
+
+    expect(wrapper.find(FormModal).prop('subtitle')).toEqual(
+      'Update the status for 2 selected items'
+    )
+  })
+
+  test('should display a subtitle if numApplicationsToUpdate == 1', () => {
+    wrapper = getWrapper({ numApplicationsToUpdate: 1 })
+
+    expect(wrapper.find(FormModal).prop('subtitle')).toBeFalsy()
+  })
+
+  test('should display a subtitle if numApplicationsToUpdate == 0', () => {
```

Comment:
this one should also be "should not display.."``
```suggestion
  test('should not display a subtitle if numApplicationsToUpdate == 0', () => {
```